### DISCUSSION TITLE
New HDR support downstream (Moonlight)

### DISCUSSION THREAD

**Discussion by dcarwin-pivotal** [LizardByte/community#158] [Sunshine]
Any guidance on getting HDR to pass all the way downstream?  It's fantastic that I don't need to turn off HDR on the PC anymore, but the image on the downstream device (Moonlight on ShieldPro / LG OLED TV) has the pale, desaturated look of a Steam screenshot taken when HDR is on.  The HDR isn't making its way to the TV yet.

**Comment by dcarwin-pivotal** [comment]
I read the other thread and now understand this is expected behavior.  Sunshine now runs when HDR is on, though it does not (yet?) stream HDR. I do think that was a good change, and thanks for contributing to Sunshine.

**Reply by roundaboutluke** [reply]
It does stream HDR. Your PC needs to have a HDR enabled monitor or dummy display dongle connected and your moonlight client needs to have the HDR setting enabled. You can confirm HDR is on as it will show HEVC HDR10 in the statistics overlay.

**Reply by dcarwin-pivotal** [reply]
I must be missing the HDR setting in Moonlight (on the Shield). I'll look for that setting. Thanks
HDR work great on the local PC.

[a comment by an automated account is not shown]
